WcaOnRails/config/routes.rb

```diff
@@ -132,6 +131,12 @@
   get 'organizer-guidelines' => 'static_pages#organizer_guidelines'
   get 'tutorial' => redirect('/files/WCA_Competition_Tutorial.pdf', status: 302)
 
+  resources :regional_organizations, only: [:new, :update, :edit]
+  get 'organizations' => 'regional_organizations#index'
+  get 'admin/regional_organizations' => 'regional_organizations#admin'
```

Comment:
nit: We prefer to use dashes (`-`) in urls, not underscores (`_`)

Reply:
But won't the URLs generated with the call to `resources` have `_` by default?

Reply:
Yeah, in any situations like that, we should be specifying `path` as in https://stackoverflow.com/questions/5334465/routes-with-dash-instead-of-underscore-in-ruby-on-rails.

Reply:
Thanks for the link! I've amended this and I will push the change now.